SU-HSDO/hs_events
# ARCH-22 ARCH-23 Copied MRC Events files

# READY FOR REVIEW

# Summary
- Abstracted MRC events to be reusable on H&S product

# Needed By (Date)
- Whenver

# Urgency
- Low

# Steps to Test

1. install standard or minimal install profile
2. install this module
3. ensure event content type exists and the fields work as expected.
4. ensure the display occurs and the path auto works. all fields aren't configured to display


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
